docker/dev/Dockerfile

```diff
@@ -1,35 +1,86 @@
-# syntax=docker/dockerfile:experimental
-FROM rust:1.47-buster as rust
+ARG RUST_BUILDER_IMAGE=ekidd/rust-musl-builder:stable
 
-ENV HOME=/home/root
+# Cargo chef plan
+FROM $RUST_BUILDER_IMAGE as planner
+WORKDIR /app
+
+# Copy dirs
+COPY Cargo.toml Cargo.lock ./
+COPY ./src ./src
+COPY ./migrations ./migrations
+COPY ./lemmy_api ./lemmy_api
+COPY ./lemmy_apub ./lemmy_apub
+COPY ./lemmy_db ./lemmy_db
+COPY ./lemmy_rate_limit ./lemmy_rate_limit
+COPY ./lemmy_structs ./lemmy_structs
+COPY ./lemmy_utils ./lemmy_utils
+COPY ./lemmy_websocket ./lemmy_websocket
+
+RUN cargo install cargo-chef --version 0.1.6
```

Comment:
You need to move this before the COPY statements above. Otherwise, any change in the Lemmy code will invalidate the Docker cache, and cargo-chef gets recompiled (which takes a while).

Reply:
I can fix.